govcd/vm.go

```diff
@@ -1487,7 +1487,29 @@ func (vm *VM) UpdateVmSpecSectionAsync(vmSettingsToUpdate *types.VmSpecSection,
 		})
 }
 
+// UpdateComputePolicyV2 updates VM compute policy and returns refreshed VM or error.
```

Comment:
Can you just add the important bit you put in `UpdateComputePolicyV2Async`. The note that it is using v2.0.0 OpenAPI endpoint.

Reply:
Done, thanks!